assets/src/application/ui/display/loadingNotice/LoadingNotice.tsx

```diff
@@ -1,33 +1,26 @@
 import React from 'react';
 import classNames from 'classnames';
-import { Spin } from 'antd';
+import { Spinner } from '@appDisplay/index';
 import { _x, sprintf } from '@wordpress/i18n';
 
 import { LoadingNoticeProps } from './types';
 import './style.scss';
 
-const LoadingNotice: React.FC<LoadingNoticeProps> = ({ children, loading, size = 'large', ...props }) => {
+const LoadingNotice: React.FC<LoadingNoticeProps> = ({ children, size = 'lg', ...props }) => {
```

Comment:
Here is the label being exposed to screen readers:
![image](https://user-images.githubusercontent.com/1477453/78865367-27aa6800-7a46-11ea-84c0-e30f9b267edc.png)